Student/src/student/StudentInfo.java

```diff
@@ -0,0 +1,73 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package student;
+
+/**
+ *
+ * @author Niaz Morshed
+ */
+public class StudentInfo {
+
+    private String name, address, phoneNo;

```

Comment:
Please declare fields individually. That way, you can add javadoc to each one describing what they represent.